packages/stores/src/mockStore.ts

```diff
@@ -583,6 +583,86 @@ const mock = (): TStores & { is_mock: boolean } => {
                 setAccountType: jest.fn(),
                 setMigratedMT5Accounts: jest.fn(),
             },
+            trade: {
+                accumulator_range_list: [],
+                active_symbols: [],
+                amount: 0,
+                barrier_1: '',
+                barrier_2: '',
+                barrier_choices: [],
+                barrier_count: 1,
+                barrier_pipsize: 1,
+                basis: '',
+                basis_list: [],
+                cached_multiplier_cancellation_list: [],
+                cancellation_duration: '',
+                cancellation_range_list: [],
+                clearContractPurchaseToastBox: jest.fn(),
+                clearPurchaseInfo: jest.fn(),
+                contract_expiry_type: '',
+                contract_purchase_toast_box: {},
+                contract_start_type: '',
+                contract_type: '',
+                contract_types_list: {},
+                currency: '',
+                duration: 0,
+                duration_min_max: {},
+                duration_unit: '',
+                duration_units_list: [],
+                expiry_date: null,
+                expiry_epoch: '',
+                expiry_time: null,
+                expiry_type: '',
+                form_components: [],
+                growth_rate: 0.01,
+                has_cancellation: false,
+                has_equals_only: false,
+                has_open_accu_contract: false,
+                has_stop_loss: false,
+                has_take_profit: false,
+                is_accumulator: false,
+                is_equal: 0,
+                is_mobile_digit_view_selected: false,
+                is_multiplier: false,
+                is_purchase_enabled: false,
+                is_trade_enabled: true,
+                is_trade_params_expanded: false,
+                is_turbos: false,
+                is_vanilla: false,
+                is_vanilla_fx: false,
+                last_digit: 9,
```

Comment:
```suggestion
                last_digit: 5,
```
@henry-deriv will it be better to put here this default value from trade-store?

Reply:
doubt it would make any difference but i'll make the change